migrate.ts

```diff
@@ -39,7 +39,7 @@ const projectJson = await getJson(
   `./source/project/${inputArgs.projKeySource}/project.json`,
 );
 
-const buildEnv: Array<any> = [];
+const buildEnv: Array<any> = [];  // To account for a potential discrepancy between source & taget projects, this array should be first populated with the environments already existing in the target project
```

Comment:
Hey, given the catch on the 409 for project creation, would this still happen? since we would break out if already created

Reply:
You're right, I wrote the comment before the fix